research-software-directory/RSD-as-a-service
# Software item mentions section

# Mentions section of software item page

Changes proposed in this pull request:

*  Add mentions section to software item page. There are slight design differences compared to legacy version due to use of different component libraries. If needed, the design can be further improved.

How to test:

*  `docker-compose up`
*  visit one of the featured software items (they have more mentions), like Xenon: http://localhost/software/xenon. At the bottom of the page you should see mentions section with one 'featured' item and number of other mentions per type.
* Click on feature item will open new web page

The software item page with the mention section should look something like in the picture below.
![image](https://user-images.githubusercontent.com/9204081/149581767-4e48e924-9689-4688-9c62-026b6aa3a28a.png)
